### DISCUSSION TITLE
BASF Data Product: integration of ODPS w/ Avro, JSONpath, and YARRRML to extend beyond API-based DPs; interest in collaboration

### DISCUSSION THREAD

**Discussion by manfred-sorg-basf** [Open-Data-Product-Initiative/v3.0#114] [General]
Hej to Finland :),
I am responsible to introduce Data Products to BASF, and I had to realize, that none of the existing specifications is complete enough for our needs. Your spec is complete in the area of data contracts etc., but in technical terms like access to databases or files, schemas, lineage, and mapping to ontologies, a lot is missing what we need. Still I like the approach of an Open Data Product Specification.
Currently I am in an aligning and convincing phase, but when I convinced the internal platforms, I'd like to cooperate with you on extending ODPS beyond OpenAPI with the aim of harmonizing our approaches.
Please find attached a Word copy - the original is currently SharePoint - for information (and for commenting if you like).
Best regards, Manfred
PS: How do you call an instance of the specification? I chose passport, but Gaia-X chose passport, too - currently I'm thinking of "Data Product Definition" ... ideas welcome.
[2024-05-13 DPS 1.0dev.docx](https://github.com/Open-Data-Product-Initiative/v3.0/files/15413305/2024-05-13.DPS.1.0dev.docx)

**Comment by kyyberi** [comment]
Hi, sorry for long delay in the response. I will take a look at the attachment and respond more thoroughly. Thank you for the patience.

**Comment by kyyberi** [comment]
Sorry for the long delay but the vacation period came in between. Would it be possible to discuss the issue in an online meeting even just briefly? If yes, could you drop an email with suitable dates and times for you at jarkko (at) vimo.fi 

Alternatively you could join our open TSC group in Whatsapp where we handle all faster pace discussion. Join from this link https://chat.whatsapp.com/IQ1ML2mvpHL3YNnlU6ir0v

**Comment by manfred-sorg-basf** [comment]
Hi Jarkko,

Yes, of course, I have a Doodle for you: https://doodle.com/meeting/participate/id/e7WgNLje
Don't mind, my vacation time will come end of August.

Best regards / Viele Grüße, Manfred (he/his)

Manfred Sorg
Data Product Coordination
Group Data Office > Data Excellence > Data and AI Powerhouse

Mobile: +49 172 7437197, Email: ***@***.***
Postal Address: BASF SE, GDB/YC - 031O30, Pfalzgrafenstraße 1, 67056 Ludwigshafen am Rhein, Germany
***@***.***
BASF SE, Registered Office: 67056 Ludwigshafen, Germany
Registration Court: Amtsgericht Ludwigshafen, Registration No.: HRB 6000
Chairman of the Supervisory Board: Kurt Bock
Board of Executive Directors:
Markus Kamieth, Chairman;
Dirk Elvermann, Michael Heinz, Anup Kothari, Stephan Kothrade, Katja Scharpwinkel

Information on data protection can be found here: https://www.basf.com/global/en/legal/data-protection-at-basf.html

Note: I support flexible working. Even when I send emails at unusual times, I never expect a response outside of your normal working hours.

From: Jarkko Moilanen ***@***.***>
Sent: Sunday, July 14, 2024 9:21 AM
To: Open-Data-Product-Initiative/v3.0 ***@***.***>
Cc: Manfred Sorg ***@***.***>; Author ***@***.***>
Subject: [EXT] Re: [Open-Data-Product-Initiative/v3.0] BASF Data Product: integration of ODPS w/ Avro, JSONpath, and YARRRML to extend beyond API-based DPs; interest in collaboration (Discussion #114)

You don't often get email from ***@***.******@***.***>. Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Sorry for the long delay but the vacation period came in between. Would it be possible to discuss the issue in an online meeting even just briefly?

-

**Comment by manfred-sorg-basf** [comment]
________________________________________________________________________________
Microsoft Teams Need help?<https://aka.ms/JoinTeamsMeeting?omkt=en-US>
Join the meeting now<https://teams.microsoft.com/l/meetup-join/19%3ameeting_MWFlOGE1ZmMtYmRjZi00NGNhLWE3MDQtNTEwNDhhNjg1MzZm%40thread.v2/0?context=%7b%22Tid%22%3a%22ecaa386b-c8df-4ce0-ad01-740cbdb5ba55%22%2c%22Oid%22%3a%22c3ede5f5-a52a-420b-b2a7-d3ed0f749587%22%7d>
Meeting ID: 358 067 649 227
Passcode: pzTy4a
________________________________
Dial in by phone
+49 69 667781609,,829791569#<tel:+4969667781609,,829791569> Germany, Frankfurt am Main
Find a local number<https://dialin.teams.microsoft.com/386c9f96-d905-4683-8317-7220a1a420cf?id=829791569>
Phone conference ID: 829 791 569#
Join on a video conferencing device
Tenant key: ***@***.***
Video ID: 128 113 830 2
More info<https://www.webex.com/msteams?confid=1281138302&tenantkey=basf&domain=m.webex.com>
For organizers: Meeting ***@***.***&messageId=0&language=en-US> | Reset dial-in PIN<https://dialin.teams.microsoft.com/usp/pstnconferencing>
________________________________________________________________________________

**Comment by manfred-sorg-basf** [comment]
Hi Jarkko,

Unfortunately our meeting didn't happen. I don't know whether the invitation reached you via this github address.
Please do me a favor and invite me (best on Fridays). I try to enable the appointment.

Best regards / Viele Grüße, Manfred (he/his)

Manfred Sorg
Data Product Coordination
Group Data Office > Data Excellence > Data and AI Powerhouse

Mobile: +49 172 7437197, Email: ***@***.***
Postal Address: BASF SE, GDB/YC - 031O30, Pfalzgrafenstraße 1, 67056 Ludwigshafen am Rhein, Germany
***@***.***
BASF SE, Registered Office: 67056 Ludwigshafen, Germany
Registration Court: Amtsgericht Ludwigshafen, Registration No.: HRB 6000
Chairman of the Supervisory Board: Kurt Bock
Board of Executive Directors:
Markus Kamieth, Chairman;
Dirk Elvermann, Michael Heinz, Anup Kothari, Stephan Kothrade, Katja Scharpwinkel

Information on data protection can be found here: https://www.basf.com/global/en/legal/data-protection-at-basf.html

Note: I support flexible working. Even when I send emails at unusual times, I never expect a response outside of your normal working hours.

**Comment by kyyberi** [comment]
Hi,

My deepest apologies, but it seems that github route is not the optimal 
as I did get invitation, but there was no date or time. Perhaps if you 
try sending it directly to me at jarkko (at ) vimo.fi

If possible, then next Friday as those days are best for you too.

Friday 26th July 3pm (UAE time zone). If this is not good for you, let 
me know which time is.

Meeting in Google Meets https://meet.google.com/ejx-pjvb-hfg

br,

Jarkko

Manfred Sorg (BASF) kirjoitti 2024-07-19 18:25:

**Comment by manfred-sorg-basf** [comment]
@kyyberi We're not lucky with meeting it seems. Let's keep this asynchronously. I mainly wanted to inform you, that we exist and that BASF is trying to follow ODPS. Secondly, I wanted to know whether you're interested in the extensions we added.
Best regards, Manfred

**Comment by kyyberi** [comment]
solved
